spec/factories/organizations.rb

```diff
@@ -45,7 +45,6 @@
 FactoryBot.define do
   factory :organization do
     division { root_division }
-    country
```

Comment:
Why did you remove country from the factory?

Reply:
it was responsible for adding new currencies since there's an association

Reply:
Ok. Thanks.